common/types/testdata/replication.go

```diff
@@ -141,4 +147,40 @@ var (
 		FailoverVersion: FailoverVersion1,
```

Comment:
hmm.  I guess this file existing is a decent argument that we do need a random-data factory :\

oh well.  additions look reasonable 👍